engine/src/flutter/tools/licenses_cpp/src/license_checker.cc

```diff
@@ -37,6 +37,8 @@ const std::array<std::string_view, 9> kLicenseFileNames = {
 const std::array<std::string_view, 2> kThirdPartyIgnore = {"pkg",
                                                            "vulkan-deps"};
 
+const char* kRootPackageName = "flutter";
```

Comment:
Can you pipe this through to `LicenseChecker::Flags`, please?  One of the big problems with the last license checker was that as people patched it they co-mingled data and logic piecemeal which made it hard to focus on fixing the actual logic.  There is already an example there for you.

Reply:
done - moved the root package name to `LicenseChecker::Flags`